stdlib/Makefile

```diff
@@ -1,46 +1,47 @@
 default: install
 
-.PHONY: default extract-pkg get-pkg clean-pkg getall install
-
 SRCDIR := $(abspath $(dir $(lastword $(MAKEFILE_LIST))))
 JULIAHOME := $(abspath $(SRCDIR)/..)
 SRCCACHE := $(abspath $(SRCDIR)/srccache)
-BUILDDIR := $(SRCCACHE)
+BUILDDIR := .
```

Comment:
Can we not cache the `Pkg-$COMMITSHA` folders in SRCCACHE instead of poisoning `/stdlib/`?

Reply:
If you want an out-of-tree build, you should do an out-of-tree build. Some Packages have a tendency to mutate their own contents (usually just a `deps/` folder), so as a whole that probably more nearly makes them build artifacts than source code. But anyways, this variable is just documenting where build artifacts must go (which must not be the same as srccache where source inputs should go); actually deciding whether something is a shared, immutable input or a build artifact is up to the rest of the Makefile.